webshoptools.php

```diff
@@ -168,3 +168,70 @@ function webshoptools_civicrm_navigationMenu(&$menu) {
   ));
   _webshoptools_civix_navigationMenu($menu);
 } // */
+
+/**
+ * Implements hook_civicrm_pre()
+ *
+ * @param $op
+ * @param $objectName
+ * @param $id
+ * @param $params
+ *
+ * @throws \CiviCRM_API3_Exception
+ */
+function webshoptools_civicrm_pre($op, $objectName, $id, &$params) {
+  if ($objectName == 'Activity' && ($op == 'create' || $op == 'edit')) {
```

Comment:
This would be a good candidate for [early return](https://pear.php.net/manual/en/standards.bestpractices.php#standards.bestpractices.returnearly).